python/src/iceberg/io/pyarrow.py

```diff
@@ -42,12 +42,12 @@ class PyArrowFile(InputFile, OutputFile):
 
     Examples:
         >>> from iceberg.io.pyarrow import PyArrowFile
-        >>> input_file = PyArrowFile("s3://foo/bar.txt")
+        >>> # input_file = PyArrowFile("s3://foo/bar.txt")
         >>> # Read the contents of the PyArrowFile instance
         >>> # Make sure that you have permissions to read/write
         >>> # file_content = input_file.open().read()
 
-        >>> output_file = PyArrowFile("s3://baz/qux.txt")
+        >>> # output_file = PyArrowFile("s3://baz/qux.txt")
```

Comment:
This change is unrelated to this PR but I noticed that since our tests run doc-strings now, this line and the line defining `input_file` was actually making a request to aws [here](https://github.com/apache/iceberg/blob/master/python/src/iceberg/io/pyarrow.py#L61) to determine the region. This just comments those lines out.